recipes/citrix-workspace/webview.js

```diff
@@ -0,0 +1,8 @@
+const _path = _interopRequireDefault(require('path'));
+
+function _interopRequireDefault(obj) {
+  return obj && obj.__esModule ? obj : { default: obj };
+}
+
+module.exports = Ferdium => {
```

Comment:
If you are not going to define notifications or use any method in the parameter, please make the parameter of the function unnamed (so that the linter is happy).